frontend/src/app/lightning/nodes-ranking/top-nodes-per-capacity/top-nodes-per-capacity.component.ts

```diff
@@ -59,6 +62,11 @@ export class TopNodesPerCapacity implements OnInit {
         })
       );
     }
+
+    this.statistics$?.subscribe((data) => {
```

Comment:
We should try stick to using RxJS observables as much as possible, instead of subscribing and setting variables. Here is an area of refactor.

Remove this code and instead merge the statistics data with the `topNodesPerChannels$` like this:


```
this.topNodesPerChannels$ = combineLatest([
        this.apiService.getTopNodesByChannels$(),
        this.statistics$
      ])
      .pipe(
          map(([ranking, statistics]) => {
          for (const i in ranking) {
            ranking[i].geolocation = <GeolocationData>{
              country: ranking[i].country?.en,
              city: ranking[i].city?.en,
              subdivision: ranking[i].subdivision?.en,
              iso: ranking[i].iso_code,
            };
          }
          ranking['totalChannels'] = statistics.latest.channel_count;
          ranking['totalCapacity'] = statistics.latest.total_capacity;
          return ranking;
        })
      );
```

Just add totalChannels and totalCapacity as optional variables to the ITopNodesPerChannels interface.

The template can now just use `node.totalChannels` and `node.totalCapacity` and drop the NgIf*.